docs/visual-testing/integrations/cypress.md

```diff
@@ -0,0 +1,185 @@
+---
+sidebar_label: Cypress
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Cypress Integration
+
+## Introduction
+
+This guide requires an existing Cypress project.<br />
+You can alternatively take a look to our [example repository](#example).
+
+Sauce Visual provides an integration with [Cypress](https://cypress.io) through a plugin that you can add to any existing Cypress project.
+
+Sauce Labs Visual introduce a new Cypress command:
+- `cy.visualCheck()`: Takes a screenshot and send it to Sauce Labs Visual for comparison.
+
+## Quickstart
+
+### Step 1: Add Sauce Visual for Cypress dependency
+
+- Install the Sauce Visual for Cypress plugin in your current project.
+
+```sh
+npm install --save @saucelabs/cypress-visual-plugin
```

Comment:
This depends a lot on user's setup. For those who have a Cypress project in a bigger project sure. For those who have a dedicated project for Cypress, it may not be effective in their CI context.

Both options are ok, IMO that's something users should be able to figure out.

Reply:
Changed it to `--save-dev`. Most customers would add this to an existing project as a dev dependency

Reply:
Changed is for all JS bindings to `--save-dev` for consistency